src/.nuget/dir.targets

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+    Finds symbol files and injects them into the package build.
+  -->
+  <Target Name="GetSymbolPackageFiles" BeforeTargets="GetPackageFiles">
+    <ItemGroup Condition="'$(SymbolFileExtension)' != ''">
+      <AdditionalLibPackageExcludes Include="%2A%2A\%2A$(SymbolFileExtension)"/>
+    </ItemGroup>
+
+    <ItemGroup>
+      <AdditionalLibPackageExcludes Include="@(LongNameFile -> '%(TargetPath)')" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <NativeWithSymbolFile Include="@(NativeBinary);
+                                     @(ArchitectureSpecificNativeFile);
+                                     @(ArchitectureSpecificNativeFileAndSymbol)">
+        <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
+      </NativeWithSymbolFile>
+      <!-- Using lib/netstandard1.0 here.  There is no TFM for this since it is a runtime itself. -->
+      <NativeWithSymbolFile Include="@(ArchitectureSpecificLibFile)">
+        <TargetPath>runtimes/$(PackageTargetRuntime)/lib/netstandard1.0</TargetPath>
+      </NativeWithSymbolFile>
+      <NativeWithSymbolFile Include="@(ArchitectureSpecificToolFile)">
+        <TargetPath>tools</TargetPath>
+      </NativeWithSymbolFile>
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(HasCrossTargetComponents)'=='true'">
+      <NativeWithSymbolFile Include="@(CrossArchitectureSpecificNativeFileAndSymbol)">
+        <TargetPath>runtimes/$(CrossTargetComponentFolder)_$(PackagePlatform)/native</TargetPath>
+      </NativeWithSymbolFile>
+      <NativeWithSymbolFile Include="@(CrossArchitectureSpecificToolFile)">
+        <TargetPath>tools/$(CrossTargetComponentFolder)_$(PackagePlatform)</TargetPath>
+      </NativeWithSymbolFile>
+    </ItemGroup>
+
+    <ItemGroup>
+      <File Include="@(NativeWithSymbolFile)" />
+      <File Include="@(LongNameFile)">
+        <IsSymbolFile>true</IsSymbolFile>
+      </File>
+    </ItemGroup>
+
+    <ItemGroup>
+      <!-- On Windows, trim ".dll" before adding ".pdb". -->
+      <WindowsNativeFile Include="@(NativeWithSymbolFile)"
+                         Condition="'%(NativeWithSymbolFile.Extension)'=='.dll' OR '%(NativeWithSymbolFile.Extension)'=='.exe'" />
+      <NativeDllFileWithoutExtension Include="@(WindowsNativeFile -> '%(RootDir)%(Directory)PDB\%(Filename)')" />
+      <WindowsSymbolFile Include="@(NativeDllFileWithoutExtension -> '%(Identity).pdb')" />
```

Comment:
No reason, was just how I took it from CoreFX's copy. Didn't notice this, thanks.